receiver/windowseventlogreceiver/README.md

```diff
@@ -88,3 +89,53 @@ Output entry sample:
 }
 ```
 
+#### Remote Configuration
+
+Single server configuration:
```

Comment:
I think it is worth to call attention that if collection of the local event log is desired a separate receiver needs to be created.

Reply:
updated